torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -166,13 +176,18 @@ def convert(self, use_path=False, *args, **kwargs):
                 del module._buffers['mask']
             delattr(module, 'mask')
 
+    def convert(self):
+        # TODO: Call the torch.ao.utils.convert in here
+        raise NotImplementedError('`convert` is not implemented. Please, use '
```

Comment:
We should have a different error message here, perhaps just say "convert is not implemented" or "use a different pruner for conversion"